src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -8,33 +8,33 @@ namespace Microsoft.Win32.SafeHandles
 {
     public abstract partial class CriticalHandleMinusOneIsInvalid : System.Runtime.InteropServices.CriticalHandle
     {
-        protected CriticalHandleMinusOneIsInvalid() : base (default(System.IntPtr)) { }
+        protected CriticalHandleMinusOneIsInvalid() : base(default(System.IntPtr)) { }
```

Comment:
Could you please undo the whitespace changes

Reply:
I just pressed "Format document" habitually. Resolved.